liota/entities/systems/dk300_system.py

```diff
@@ -0,0 +1,8 @@
+from system import System
+from liota.utilities.utility import systemUUID
+
+
+class Dk300System(System):
+
+    def __init__(self, name):
+        super(Dk300System, self).__init__(name=name, entity_id=systemUUID().get_uuid(name))
```

Comment:
Something has to be passed from the child class as it is a parameter. So better to generate the UUID here.